src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2504,6 +2504,8 @@
   "com.minecolonies.coremod.questobjectives.buildbuilding.cumulative.existing": "I am still waiting for you to reach %s totaling %d levels!",
   "com.minecolonies.coremod.questobjectives.buildbuilding.progress": "%d/%d %s buildings built",
   "com.minecolonies.coremod.questobjectives.buildbuilding.progress.cumulative": "%d/%d totaling levels of %s buildings built",
+  "com.minecolonies.coremod.questobjectives.research.progress": "%s is not researched yet!",
+  "com.minecolonies.coremod.questobjectives.research":  "I am still waiting for you to research %s at the university!",
```

Comment:
maybe more of a "I am waiting for the research %s to be completed at the university"